cli.js

```diff
@@ -1,17 +1,35 @@
 const { Command } = require('commander');
 const fs = require('fs');
+const path = require('path');
 const exportModule = require('./exportModule.js');
 const logger = require('./winston.js');
 
 const program = new Command();
 program.version('0.0.1');
 function data() {
-  program.requiredOption('--elmJSON <file>', 'elm json to turn to synthea module').parse(process.argv);
-  const contents = JSON.parse(fs.readFileSync(program.elmJSON, 'utf8'));
-  return contents;
+  program
+    .requiredOption('--elmJSON <file>', 'elm json to turn to synthea module')
+    .option('--library <library>', 'directory of dependent libraries');
+  program.parse(process.argv);
+  const returns = {};
+  const mainFile = fs.readFileSync(program.elmJSON, 'utf8');
+  returns.mainLibrary = JSON.parse(mainFile);
+  if (program.library !== undefined) {
+    const directoryPath = path.join(__dirname, program.library);
+    returns.dependencies = [];
+    try {
+      const files = fs.readdirSync(directoryPath);
+      files.forEach((file) => {
+        const library = fs.readFileSync(`${directoryPath}/${file}`, 'utf8');
+        returns.dependencies.push(JSON.parse(library));
+      });
+    } catch (err) {
+      console.log(err);
+    }
+  }
+  return returns;
 }
 
+logger.info(`name of file: ${data().mainLibrary.library.identifier.id}.json`);
 const moduleJSON = exportModule(data());
-logger.info(`name of file: ${data().library.identifier.id}.json`);
-
-fs.writeFileSync(`${data().library.identifier.id}.json`, JSON.stringify(moduleJSON));
+fs.writeFileSync(`${data().mainLibrary.library.identifier.id}.json`, JSON.stringify(moduleJSON));
```

Comment:
Small suggestion, but i would pull out the value of `data()` into a variable and then use that in the log statements, exportModule call, and file writing. Just so we don't actually call the function 3 different times with no changes.